src/sentry/lang/javascript/processor.py

```diff
@@ -187,24 +187,48 @@ def fetch_release_file(filename, release):
     cache_key = 'releasefile:v1:%s:%s' % (
         release.id,
         md5(filename).hexdigest(),
-    )
+    ),
+
+    filename_path = None
+    try:
+        # Reconstruct url without protocol + host
+        # e.g. http://example.com/foo?bar => ~/foo?bar
+        parsed_url = urlparse(filename)
+        filename_path = '~' + parsed_url.path
+        if parsed_url.query:
+            filename_path += '?' + parsed_url.query
+    except Exception:
```

Comment:
I just said the same thing. There's no case when this would raise from my understanding. 

--
Matt Robenolt
@mattrobenolt




On Thu, Jun 16, 2016 at 2:31 AM -0700, "Armin Ronacher" <notifications@github.com> wrote:












In src/sentry/lang/javascript/processor.py:
> @@ -187,24 +187,48 @@ def fetch_release_file(filename, release):
>      cache_key = 'releasefile:v1:%s:%s' % (
>          release.id,
>          md5(filename).hexdigest(),
> -    )
> +    ),
> +
> +    filename_path = None
> +    try:
> +        # Reconstruct url without protocol + host
> +        # e.g. http://example.com/foo?bar => ~/foo?bar
> +        parsed_url = urlparse(filename)
> +        filename_path = '~' + parsed_url.path
> +        if parsed_url.query:
> +            filename_path += '?' + parsed_url.query
> +    except Exception:



The link does not go anywhere @mattrobenolt 



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.